pom.xml

```diff
@@ -135,9 +195,25 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-enforcer-plugin</artifactId>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-release-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+      </plugin>
     </plugins>
   </build>
 
+  <distributionManagement>
+    <snapshotRepository>
```

Comment:
Is there a reason to not also include the release repository here? That will be needed for performing releases.

Reply:
not sure here, i'll look into it. neither of the exemplars (e.g. https://bitbucket.org/simpligility/ossrh-pipeline-demo/src/master/pom.xml ) mentioned a release repository in this section - i wonder if the release repo is understood in the nexus sonatype plugin

Reply:
i haven't been able to track this down, but the actual release is technically outside the scope of this card (SNAPSHOT deploys are covered). i think we can put this off until it is needed - it is an easy add in the configuration.